amida-tech/indaba-client
# Feature/INBA-495 PM User Removal Checks

#### What's this PR do?
Displays a prompt with language that depends on the state of a user's data before removing the user from a project on the users tab of pm project view.

#### Related JIRA tickets:
[INBA-495](https://jira.amida-tech.com/browse/INBA-495)

#### How should this be manually tested?
1. Create a project with a user in each of these conditions:
   1. Has no tasks assigned
   1. Has tasks assigned but has not answered questions on any of them
   1. Has answered questions on an assigned task
1. Delete each user and check that the modal language is [correct based on the data state](https://paper.dropbox.com/doc/Indaba-Conditional-Limits-ublG6ZqtvItRL1LL924Zd#:uid=093953831492899346329275&h2=Conditional-Delete)

#### Any background context you want to provide?
Make sure [INBA-496](https://github.com/amida-tech/greyscale/pull/392) is merged or your backend instance is on that branch to prevent errors during the data check

This branch sits on #246, review that first for a smaller changeset

#### Screenshots (if appropriate):
